### PR TITLE
Database/update db #96

### PR DESCRIPTION
**Pull request by silent-po2**
Updated DB to account for new stakeholder requirements

### CODE REVIEW

File: Server/src/database.ts [hunk 1]
```diff
@@ -12,399 +12,507 @@ import { Subcategory } from './models/entities/Subcategory';
...
+    await connection.manager.save(datapoint11);
+
+    console.log("All data entered");
+
+    // End element insertion
+
+    /**
+     * Testing finding an element
+     */
+    // let savedAnime = await connection.manager.find(AnimeRank, { title: "Promised Neverland"});
```
Comment (JeremiahTee) [inline Server/src/database.ts:485]: Are these commented out snippets of code for testing necessary? Do you think we'll need them in production eventually (master)?

Or have you kept them simply for convenience, that you may use them in the future when updating the DB?

Comment (silent-po2) [reply Server/src/database.ts:485]: Yes these are very much needed. While the terminology here is for testing, specifically they will form the basis of multiple queries to be created in #53. As #53 progresses, these will be removed

**Comment by JeremiahTee** [comment]
Reviewed the changes during a peer programming session. I also ran the changes on my end, using Windows 10 & latest docker. No issues. Approved

**Review by JeremiahTee — APPROVED** [review]

**Review by TiraFire — APPROVED** [review]
trust you, looks good!

**Pull request merged**